test/runtests.jl

```diff
@@ -560,11 +561,7 @@ r = internalrequest(HTTP.Request("GET", "/static/sample.html"))
 @test r.status == 200
 @test text(r) == file("content/sample.html") |> text
 
-r = internalrequest(HTTP.Request("GET", "/multiply/a/8"))
-@test r.status == 500
```

Comment:
this test seemed to be duplicated so I deleted one of them, though maybe I misunderstood what it is doing